src/js/components/Drop/README.md

```diff
@@ -271,6 +271,22 @@ string
     xlarge
     string
 }
+{
+  corners: [{
+  corner: 
+    top-left
+    top-right
+    bottom-left
+    bottom-right,
+  size: 
+    xsmall
+    small
+    medium
+    large
+    xlarge
+    string
+}]
+}
```

Comment:
Readme files should only include a link to the documentation. See other components for examples

Reply:
Ok I'll take a look. Thanks for the feedback.